examples/complete/main.tf

```diff
@@ -25,6 +25,11 @@ module "key_protect" {
   ]
 }
 
+resource "ibm_iam_api_key" "api_key" {
+  name        = "${var.prefix}-api-key"
+  description = "created for secrets-manager-secret complete example"
+}
```

Comment:
Why are we creating a user apikey in the example? The example already takes in an apikey via the var.ibmcloud_api_key input